cirq-core/cirq/ops/dense_pauli_string.py

```diff
@@ -614,13 +614,13 @@ def _as_pauli_mask(val: Iterable[cirq.PAULI_GATE_LIKE] | np.ndarray) -> np.ndarr
 
 
 def _attempt_value_to_pauli_index(v: cirq.Operation) -> tuple[int, int] | None:
```

Comment:
@ToastCheng Note that this could easily be expanded now to return a `list[tuple[int, int]]` if v has multiple qubits, and allow DensePauliString to multiply and commute with parity gates etc on LineQubits too. I think that would be a separate PR, but would be a straightforward followup.

Reply:
Thanks @daxfohl! I saw you had another PR #7643 to replace this function, I assume there is no need to expand this function?

Reply:
Yeah I went ahead and took it.